pci/src/vfio.rs

```diff
@@ -1662,8 +1662,12 @@ impl VfioPciDevice {
                                 user_memory_region.host_addr,
                             )
                             .map_err(|e| {
-                                VfioPciError::DmaMap(e, self.device_path.clone(), self.bdf)
-                            })?;
+                                error!(
```

Comment:
I think this should be a `warn!` - `error!` is reserved for more fatal errors. Adjust the error message to something like:                                     

warn!("Unable to map VFIO device region for {} - P2P operations will not be available", self.bdf)

Reply:
I had it at `warn!` initially, but since I expect this code path to trigger the (non-fatal) `error!` message from the `unmap` attempt on teardown later, I thought it may make more sense to use the same severity here. I can downgrade to a warning of course if you think this is still obvious enough in the logs. When you hit this message, you probably want to double-check the setup.

Should the corresponding `unmap` stay at error level? https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/pci/src/vfio.rs#L1680-L1685 I feel like it should be error if the mapping succeeded, a warning otherwise. But implementing that check is a bit overkill.